sdcm/nemesis.py

```diff
@@ -4417,6 +4423,106 @@ def disrupt_add_remove_mv(self):
         finally:
             self.unset_current_running_nemesis(cql_query_executor_node)
 
+    def disrupt_toggle_audit(self, store: AuditStore):
```

Comment:
this is problematic, we don't support very well parameter to disrupt function, i.e. it would always gonna be empty (when used in any Nemesis class, beside ToggleAuditNemesis.

so or we set a default value here to syslog, or we refactor ro multiple disrupt functions calling other function

Reply:
refactor to multiple disrupt functions is better - different nemesis was planned for table so later we would need to rename and break ES stats and possibly some other areas.